libs/designer-ui/src/lib/editor/base/index.tsx

```diff
@@ -163,6 +164,8 @@ export const BaseEditor = ({
   };
 
   const id = useId('msla-described-by-message');
+  const TextPlugin = toolbar ? RichTextPlugin : PlainTextPlugin;
```

Comment:
Open question: Is `toolbar` here the best way to detect if we're in rich text mode?

Reply:
Looks like this is what's used on line 135, so should be safe.

Reply:
yeah this is fine, though I'm gonna create an item to move the toolbar prop into something a bit more inline with the HTML editor